scss/_chat.scss

```diff
@@ -470,10 +491,31 @@
         word-break: break-word;
       }
     }
+    .message-own {
+      background: transparent;
+    }
     &.message-own .message-content .bbcode {
       background: transparent;
     }
   }
+  .message-highlight {
+    border-left: solid 2px
```

Comment:
Not a big fan of setting rules with !important tags compared to increasing the specificity levels. That tends to get really annoying when exceptions to those rules are created.